Sample Data/ASIM/SonicWallFirewall/SonicWall_Firewall_WebSession_IngestedLogs.csv

```diff
@@ -0,0 +1,1294 @@

```

Comment:
@jaimeesc 
Please keep the sample logs here: https://github.com/Azure/Azure-Sentinel/tree/master/Sample%20Data/ASIM
Not in SonicWall folder. Also, we need to move network sample log to above location as well. I missed seeing this in the last PR.